<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OMPClass</class>
 <widget class="QMainWindow" name="OMPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OMP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="3">
     <widget class="View" name="graphicsView"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="gbAddItem">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="bRect">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>üü•</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bEllipse">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>üü¢</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bTriangle">
         <property name="text">
          <string>üî∫</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="gbModItem">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="bConnect">
         <property name="text">
          <string>üìè</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bDrag">
         <property name="text">
          <string>üñê</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bDel">
         <property name="text">
          <string>‚ùå</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header>view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
